psst-gui/src/ui/preferences.rs

```diff
@@ -228,7 +228,6 @@ fn general_tab_widget() -> impl Widget<AppState> {
                         |usize: &usize| usize.to_string(),
                     )),
                 )
-                .padding((theme::grid(1.5), 0.0))
```

Comment:
Were these lints or are they fixing a UI issue?

Reply:
Just fixing a UI issue, they really shouldn't have been there given that we don't left pad anything else.